### PR TITLE
Ppodae

### PR DESCRIPTION
**Pull request by scionalu**
## About The Pull Request

Adds Ppodae
Cleaned up most things from last PR

## Why It's Good For The Game

Adds a new abnormality

## Changelog
:cl:
Removed unused lines
Fixed damage/defense stats of ego gear and ppodae
Changed basic attack to aoe
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+		if(length(parts))
+			var/obj/item/bodypart/bp = pick(parts)
+			bp.dismember()
+			bp.forceMove(ppodae_location) // Teleports limb to containment
+			QDEL_NULL(src)
+			// Taken from eldritch_demons.dm
+	return Smash(target)
+
+//AoE attack taken from woodsman
+/mob/living/simple_animal/hostile/abnormality/ppodae/proc/Smash(target)
```
Comment (scionalu) [inline code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm:67]: Basically just a 2x3 area

File: code/game/objects/items/ego_weapons/teth.dm [hunk 2]
```diff
@@ -149,3 +149,14 @@
...
+
+/obj/item/ego_weapon/cute
+	name = "SO CUTE!!!"
+	desc = "One may think, 'How can a weapon drawn from such a cute Abnormality be any good?' \
+		However, the claws are actually quite durable and sharp."
+	icon_state = "cute"
+	force = 13
+	attack_speed = 0.5
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
```
Comment (EgorDinamit) [inline code/game/objects/items/ego_weapons/teth.dm:161]: Red damage is the default, so this is obsolete.

File: code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+	can_breach = TRUE
+	start_qliphoth = 2
+	vision_range = 14
+	aggro_vision_range = 20
+	stat_attack = HARD_CRIT
+	var/smash_length = 2
+	var/smash_width = 1
+	var/can_act = TRUE
+
+	var/turf/ppodae_location
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm:32]: This is totally not needed. You can just get the landmark's location from datum_reference.

File: code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+	var/smash_damage = rand(6, 10)
+	for(var/turf/T in area_of_effect)
+		new /obj/effect/temp_visual/smash_effect(T)
+		for(var/mob/living/L in T)
+			if(faction_check_mob(L))
+				continue
+			if (L == src)
+				continue
+			L.apply_damage(smash_damage, RED_DAMAGE, null, L.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)
+			playsound(get_turf(src), 'sound/effects/tableslam.ogg', 75, 0, 5)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm:158]: Might want to get sounds from LC instead.

**Comment by scionalu** [comment]
I'll fix stuff tomorrow

File: code/game/objects/items/ego_weapons/teth.dm (hunk 2 above)
Comment (scionalu) [reply code/game/objects/items/ego_weapons/teth.dm:161]: All ego weapons (I didn't check guns though) still puts it in even if they were red
Honestly at this point it's more like a theme
I'll erase it if you still don't like it though

File: code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm (hunk 4 above)
Comment (scionalu) [reply code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm]: Done

File: code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm (hunk 3 above)
Comment (scionalu) [reply code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm]: Done

**Pull request merged**